SignallingWebServer/package.json

```diff
@@ -8,47 +8,31 @@
         "build": "tsc",
         "rebuild": "npm run clean && npm run build",
         "start": "node ./dist/index.js --serve --console_messages verbose --log_config --https_redirect",
-        "lint": "eslint ./src --ext .js,.jsx,.ts,.tsx",
+        "lint": "eslint src",
         "test": "echo \"Error: no test specified\" && exit 1",
         "watch": "nodemon -V -d 3 --watch src --watch ../Signalling/dist -e \"ts,js,mjs,cjs,json\" --exec \"npm run build && node ./dist/index.js --serve --console_messages verbose --log_config --https_redirect --player_port 1025\" --http_root www",
         "develop": "cd ../ && npm run build:all:cjs && cd SignallingWebServer && concurrently -k \"cd ../Common && npm run watch\" \"cd ../Signalling && npm run watch\" \"cd ../Frontend/library && npm run watch\" \"cd ../Frontend/ui-library && npm run watch\" \"cd ../Frontend/implementations/typescript && npm run watch\"  \"npm run watch\""
     },
     "author": "Epic Games",
     "license": "MIT",
     "devDependencies": {
-        "@types/express": "^4.17.21",
-        "@types/node": "^20.11.13",
-        "@types/ws": "^8.5.10",
-        "@typescript-eslint/eslint-plugin": "^6.21.0",
-        "@typescript-eslint/parser": "^6.21.0",
-        "concurrently": "^9.1.2",
-        "eslint": "^8.56.0",
-        "eslint-config-prettier": "^9.1.0",
-        "eslint-plugin-prettier": "^5.2.1",
-        "eslint-plugin-tsdoc": "^0.2.17",
-        "nodemon": "^3.0.3",
-        "prettier": "3.3.3",
-        "ts-node": "^10.9.2",
-        "typedoc": "^0.25.8",
-        "typedoc-plugin-markdown": "^3.17.1",
-        "typescript": "^5.3.3"
+        "@types/express": "^5.0.0",
+        "eslint": "^9.20.0",
+        "eslint-config-prettier": "^10.0.1",
+        "eslint-plugin-prettier": "^5.2.3",
+        "eslint-plugin-tsdoc": "^0.4.0",
+        "jsonc": "^2.0.0",
```

Comment:
jsonc seems to be used in source in `SignallingWebServer\src\Utils.ts` it should be a dep